app/models/trainee.rb

```diff
@@ -267,24 +269,30 @@ def can_apply_for_bursary?
     training_route == TRAINING_ROUTE_ENUMS[:early_years_postgrad] || bursary_amount.present?
   end
 
+  def new_and_empty?
+    non_section_fields = %w[created_at updated_at dttp_update_sha progress slug training_route id provider_id]
+
+    # If the trainee has section related fields set the value digest will be a long string containing all the current values
+    trainee_values(non_section_fields).join(",") == "draft"
+  end
+
 private
 
-  def value_digest
+  def value_digest(exclude_list)
     # this returns a comma separated string of values from this object and it's associations
     # we use this to determine if we need to update DTTP. We use values only and exclude nils to avoid
     # sending updates when we add a field to the schema.
-
-    exclude_list = %w[created_at updated_at dttp_update_sha progress]
-
-    trainee_values = serializable_hash.reject { |k, _v| exclude_list.include?(k) }.values.compact
-
     (
-      trainee_values + [degrees, nationalities, disabilities].flat_map do |assoc|
+      trainee_values(exclude_list) + [degrees, nationalities, disabilities].flat_map do |assoc|
         assoc.map(&:serializable_hash).flat_map(&:values).compact
       end
     ).join(",")
   end
 
+  def trainee_values(exclude_list = [])
```

Comment:
Just a suggestion, but this calls to this method might read better with a named argument `trainee_values(exclude: non_section_fields)`. 

```suggestion
  def trainee_values(exclude: [])
```